README.md

```diff
@@ -1,50 +1,95 @@
+ * Install a recent version of
+   [Docker](https://docs.docker.com/installation/#installation)
 
-Make sure that docker-machine is create and running first.
 
-```bash
-docker-machine create --driver virtualbox dev
-```
-
-Run `eval $(docker-machine env default)` in each shell you'll use to interact with Docker.
-
-### Getting Started
+### Windows or Linux
 
-Clone this repository and `cd` into the directory where you cloned the repo. Then:
-
-1. Customize your credentials and other settings. The `env`
-   settings require knowing your Rackspace Cloud account info
-   and also where you intend to clone the control repo, for example.
+ 1. Install [docker-compose](https://docs.docker.com/compose/install/)
+ 1. Create a `docker-machine dev` instance:
     ```bash
-    cp env.example env
-    ${EDITOR} env
+    docker-machine create --driver virtualbox dev
     ```
-1. Launch the services.
-   ```bash
-   script/up
-   ```
+ 1. Run `eval $(docker-machine env default)` in each shell that you'll
+    use to interact with Docker.
+
+
+## Edit the env file
+
+Clone this Deconst Integrated repository and change to the directory
+where you cloned the repo. Customize your credentials and other
+settings in the `env` file. Setting the variables correctly require
+knowing your Rackspace Cloud account info and also where you intend to
+clone the control repo.
+
+   1. Copy the example file to `env`:
+      ```bash
+      $ cp env.example env
+      ```
+   1. Edit the `env` file in a text editor and change it as
+      appropriate for your environment.
+   1. Create a random API key. For example:
+      ```bash
+      $ hexdump -v -e '1/1 "%.2x"' -n 40 /dev/random
+      50d1606d4bd6bd1a5f2adefdcae603deff9b012a164cb8bf0b68caf3638d8e868
+      ```
+   1. Paste the key between the quotes on this line in the
+      `env` file:
+      ```bash
+      # The content service's administrative API Key. Used for write actions.
+      # Set this to something arbitrary and "random". While you're running locally it doesn't need to
+      # be anything particularly difficult.
+      export ADMIN_APIKEY="50d1606d4bd6bd1a5f2adefdcae603deff9b012a164cb8bf0b68caf3638d8e868""
+      ```
+   1. Set the domain name of the published documentation site:
+      ```bash
+      # Set this to the domain name of the site you're interested in.
+      export PRESENTED_URL_DOMAIN=deconst.horse
+      ```
+   1. Set the location of your `nexus-control` repository. For
+      example:
+      ```bash
+      # Set this to a path to a control repository on your local
+      # machine to preview local changes to a control repository.
+      unset CONTROL_REPO_HOST_PATH
+      export CONTROL_REPO_HOST_PATH="/Users/writer1/nexus-control"
+      ```
+   1. Set _presenter_ developer mode to true:
+      ```bash
+      # "true" or "false", whether the presenter should load assets directly
+      # from the control repo, or via the assets container export
+      PRESENTER_DEVMODE=true
```

Comment:
This change will only work for the `nexus-control` build as it's implemented in the control repo, not the presenter codebase. As this should be universal since it's in the deconst org, instead, the directions should be to run the `add-assets` code as a general info piece, and the adjustment to add the devmode would be defined in the readme for the nexus-control repo (there's an env file there for use with integrated, actually).

Reply:
changed.